src/styles/media.scss

```diff
@@ -0,0 +1,17 @@
+@media (max-width: 1023px) and (min-width: 768px) {
```

Comment:
since it lives in the main.scss you should delete this file to avoid duplication